### PR TITLE
utils/perf.py: copy `_parse_perf_script` and `_collapse_stack` from `profilers/perf.py`

### PR DESCRIPTION
**Pull request by roi-granulate**
## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the relevant documentation.
- [ ] I have added tests for new logic.

### CODE REVIEW

File: gprofiler/utils/perf.py [hunk 1]
```diff
@@ -44,3 +69,51 @@ def valid_perf_pid(pid: int) -> bool:
     try and look up the processes related to them.
     """
     return pid not in (0, -1)
+
+
+def _collapse_stack(comm: str, stack: str, insert_dso_name: bool = False) -> str:
```
Comment (Jongy) [inline gprofiler/utils/perf.py:74]: Exported stuff - don't prefix with `_`

File: gprofiler/utils/perf.py [hunk 2]
```diff
@@ -44,3 +69,51 @@ def valid_perf_pid(pid: int) -> bool:
...
+        # append kernel annotation
+        elif "kernel" in dso or "vmlinux" in dso:
+            sym += "_[k]"
+        elif insert_dso_name:
+            sym += f" ({dso})"
+        funcs.append(sym)
+    return ";".join(funcs)
+
+
+def _parse_perf_script(script: Optional[str], insert_dso_name: bool = False) -> ProcessToStackSampleCounters:
```
Comment (Jongy) [inline gprofiler/utils/perf.py:95]: Same here - remove `_`

**Review by Jongy — CHANGES_REQUESTED** [review]
Done reviewing

File: gprofiler/utils/perf.py (hunk 1 above)
Comment (roi-granulate) [reply gprofiler/utils/perf.py:74]: IDM, I wanted this PR to be a copy (except imports let's say)

**Review by Jongy — APPROVED** [review]

**Pull request merged**